tests/test-thread.lua

```diff
@@ -62,14 +62,16 @@ return require('lib/tap')(function (test)
     local delay = 100
     uv.update_time()
     local before = uv.now()
-    local args = {delay, 'string', nil, false, 5, "helloworld"}
+    local args = {delay, 'string', nil, false, 5, 3.14, "helloworld"}
     local unpack = unpack or table.unpack
-    uv.new_thread({stack_size=0}, function(delay,s,null,bool,five,hw)
+    uv.new_thread({stack_size=0}, function(delay,s,null,bool,five,pi,hw)
       assert(type(delay) == "number")
       assert(type(s) == "string")
       assert(null == nil)
       assert(bool == false)
       assert(five == 5)
+      assert(math.type(five) == 'integer')
```

Comment:
```suggestion
      if math.type then
        assert(math.type(five) == 'integer')
      end
```

Reply:
Fixed in commit 0d141bff9dcb3f05c2e19b40af48cd9f4448e8db